week3/project/ecommerce3/src/components/FavoriteBtn.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useContext } from "react";
+import { FavoriteContext } from "../context/FavoriteContext";
+import regularHeart from "../assets/heart-regular.svg";
+import activeHeart from "../assets/heart-solid.svg";
+
+import "./components-css/btn.css";
+
+const FavoriteBtn = (id) => {
```

Comment:
you could name the parameter like that
```js
const FavoriteBtn = ({itemId}) => {
```
then you can use this variable directly without renaming it in the next line